Sources/LanguageServerProtocol/SupportTypes/CompletionItem.swift

```diff
@@ -71,6 +91,21 @@ public struct CompletionItem: Codable, Hashable {
     self.insertTextFormat = insertTextFormat
     self.kind = kind
     self.deprecated = deprecated
+    self.data = data
+  }
+  
+  enum CodingKeys: String, CodingKey {
```

Comment:
Maybe a stupid question but why do we need to specify the coding keys here? AFAICT we only provide it in other types if we are customizing the decoding initializer or mapping the coding keys to different strings and I think none of this is happening here.

Reply:
I originally thought `textDocument` would be encoded, but I just tried removing `CodingKeys` and it actually isn't encoded, so I'll remove this.